cassandra-dac/src/main/java/org/sunbird/cassandra/CassandraOperation.java

```diff
@@ -51,6 +51,16 @@
    */
   public Response deleteRecord(String keyspaceName, String tableName, String identifier);
 
+  /**
+   * @desc This method is used to delete record in cassandra db by their primary composite key
+   * @param keyspaceName String (data base keyspace name)
```

Comment:
Change comments:
keyspaceName Keyspace name
tableName Table name
compositeKeyMap Column map for composite primary key

Change return type to void.